src/init.cpp

```diff
@@ -1507,6 +1509,8 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
             fLoaded = true;
         } while(false);
 
+        if (strLoadError.empty() && fRequestShutdown) break;
+
```

Comment:
If there's an error, we'll handle that below, so don't break.

Reply:
What's the difference between fRequestShutdown and ShutdownRequested()? We should probably only use one or the other.